src/ctl/ctl.c

```diff
@@ -78,16 +80,34 @@ char *Strdup(const char *s) {
     return p;
 }
 
+int umf_ctl_get(const char *name, void *ctx, void *arg) {
+    return ctl_query(NULL, ctx, CTL_QUERY_PROGRAMMATIC, name, CTL_QUERY_READ,
+                     arg);
+}
+
+int umf_ctl_set(const char *name, void *ctx, void *arg) {
+    return ctl_query(NULL, ctx, CTL_QUERY_PROGRAMMATIC, name, CTL_QUERY_WRITE,
+                     arg);
+}
+
+int umf_ctl_exec(const char *name, void *ctx, void *arg) {
+    return ctl_query(NULL, ctx, CTL_QUERY_PROGRAMMATIC, name,
+                     CTL_QUERY_RUNNABLE, arg);
+}
+
 /*
  * ctl_find_node -- (internal) searches for a matching entry point in the
  *    provided nodes
  *
+ * Name offset is used to return the offset of the name in the query string.
  * The caller is responsible for freeing all of the allocated indexes,
  * regardless of the return value.
  */
 static const struct ctl_node *ctl_find_node(const struct ctl_node *nodes,
                                             const char *name,
-                                            struct ctl_index_utlist *indexes) {
+                                            struct ctl_index_utlist *indexes,
```

Comment:
could you typedef clt_index_utlist -> umf_clt_index_utlist and get rid of "struct" ? 
same for ctl_node

this would be more consistent with the rest of code

Reply:
Done